### DISCUSSION TITLE
Support for project file of version 4.2 and older has ended

### DISCUSSION THREAD

**Discussion by DamoiseauxD** [Open-Systems-Pharmacology/Forum#1053] [Questions & Problems]
Hi all,

When I tried to open an old file (.pkprj) I got the following message. However, I am unable to find any downloads for a PK-Sim version 5.5 or earlier (the oldest release I could find was 7.0.0). Is there another way to convert the old file or can anyone tell me where I can download version 5.5 or earlier.

![image](https://user-images.githubusercontent.com/58684068/145545686-0e06bab2-b4a3-4677-91ec-0d619fb009ee.png)

Thanks in advance!

David

**Comment by StephanSchaller** [comment]
@DamoiseauxD , yes, there likely is, and I am sure someone can help you.

(I can't but) An intermediate solution might be to look ate the files (which are xml-based if not encoded) with e.g. notepad++
They are quite well structure so it might allow you to extract all relevant information for re-implementation in V10.

**Comment by msevestre** [comment]
@DamoiseauxD I don't think this will be possible. Before v7, PK-Sim was not freeware and required a license key. You will not be able to start an older version without it
